test/visual-diff/visual-tests.json

```diff
@@ -70,6 +70,16 @@
       "url": "examples/visual-tests/amp-by-example/amp-by-example.html",
       "name": "Amp By Example"
     },
+
+    {
+      "url": "examples/visual-tests/amp-list/amp-list.amp.html",
+      "name": "AMP List and Mustache",
+      "loading_complete_css": [
+        "#list1",
```

Comment:
Curious: Does this actually end up waiting? I thought css classes begin with a period, like `.list1`.

Reply:
I see from the HTML files that it's an `id`, not a `class`. I believe capybara needs you to use a class name here for it to wait until the element appears. If not, there might be flakiness in this test.

Reply:
All other appear to be CSS selectors. So I assume it will lookup ID CSS selector just as well?

Reply:
I changed to classes to avoid flakiness, but I'd still like to know if ID-selectors supported.